src/server.ts

```diff
@@ -264,21 +385,20 @@ export class SubscriptionServer {
                 }
 
                 // NOTE: This is a temporary code to identify if the request is a real subscription or only a query/mutation.
-                // As soon as subscription manager starts handling complete function from the observable calling  the
-                // callback function with (null, null) this can be replaced
+                // As soon as we completely drop the subscription manager and starts handling complete function
+                // from the observable, calling the callback function with (null, null), this can be replaced
                 if (!isSubscription) {
```

Comment:
this can be removed, as you should always get observable complete.
also, GQL_COMPLETE should be hooked up to observable compelete